lib/api/include/data_object_finalize.h

```diff
@@ -18,13 +18,18 @@ extern "C" {
 /// desired modifications is required in the form of a JSON structure. Each replica can also
 /// have "file_modified" key which holds a set of key-value pairs for the file_modified plugin operation.
 ///
-/// The file_modified field is a boolean indicating whether the file_modified plugin operation
+/// The trigger_file_modified field is a boolean indicating whether the file_modified plugin operation
 /// should be called after the data object has been finalized.
+///
+/// irodsAdmin is a special keyword in iRODS to indicate that the operation requires elevated
+/// privileges. When present and set to true, the API will honor elevated privileges. If the user
+/// is not authorized to use elevated privileges, an error will be returned.
 /// \endparblock
 ///
 /// \p json_input must have the following JSON structure:
 /// \code{.js}
 /// {
+///     "irodsAdmin": <bool>,
```

Comment:
This is just what `ADMIN_KW` is set to. I'm fine with using `irods_admin` if that's preferable. I'd probably like to introduce a new keyword though. Thoughts?

Reply:
if `irodsAdmin` is not special, then i vote for the underscore'd version.

i'm interested in the new keyword you're thinking about as well. it may be better than the other two.

Reply:
Oh, I was just going to make a new keyword for `irods_admin` to which all of the code could refer just to avoid hard-coding the string everywhere.

Reply:
I've replaced the use of `ADMIN_KW` with `irods_admin` for the key.